design-documents/static-checks-for-templates.md

```diff
@@ -0,0 +1,27 @@
+### Overview
+
+At the time of this writing only 85% of the `*.phtml` files in Magento core are being statically 
+tested due to `@codingStandardsIgnore` present in over 700 templates. There is a current security initiative 
+that requires a large number of templates to be modified and they need to be covered by static checks moving 
+forward as part of the Acceptance Criteria of the initiate. 
+
+However, due to the lack of current static template checks the codebase has thousands of static template errors and will require many 
+hours to address every issue in every file. Additionally, many of the tests are not reasonable, checking incorrectly, or simply gratuitous for `*.phtml` templates. 
+
+
+### Design
+
+To address this issue for the immediate efforts as well as for all developers in the future I am proposing to remove the following sniffs
```

Comment:
Does this mean that you also propose to start removing `@codingStandardsIgnore` which would require fixing existing errors?

Reply:
Yes, and all files affected by the security efforts (hundreds of phtml files) will be cleaned up as part of our initial merges in the next couple weeks.

Reply:
I may be wrong, but I believe there is already a backlog item to remove all `@codingStandardsIgnore` annotations.